coda/coda_mdstore/tests/test_views.py

```diff
@@ -678,6 +678,21 @@ def test_response_content_has_topFiles(self, rf):
                 b'https://coda/data/file01.txt') in response.content
         assert response.status_code == 200
 
+    def test_response_with_bagfiles(self, rf):
+        """"Test response content with bagfiles kwargs."""
+        self.getFileHandle.return_value.url = 'https://coda/testurl'
+        # Mock what gets read from the manifest file.
+        self.getFileHandle.return_value.readline.side_effect = [
+            b'192e635b17a9c2aea6181f0f87cab05d  data/file01.txt',
+            b'18b7c500ef8bacf7b2151f83d28e7ca1  data/file02.txt',
+            b'']
+        bag = FullBagFactory.create()
+        request = rf.get('/')
+        response = views.bagURLList(request, bag.name, bagfiles='bagfiles')
+        assert b'<a' in response.content
```

Comment:
I don't really think this assertion by itself adds much. Instead, why don't we add the full <a> tag (opening and closing) to the next two assertions so they each check not only that they URL is present, but also that it's in link form?